builder/ebs/builder_acc_test.go

```diff
@@ -770,6 +770,40 @@ func TestAccBuilder_EbsKeyPair_rsaSHA2OnlyServer(t *testing.T) {
 	acctest.TestPlugin(t, testcase)
 }
 
+//go:embed test-fixtures/unlimited-credits/burstable_instances.pkr.hcl
+var testBurstableInstanceTypes string
+
+func TestAccBuilder_EnableUnlimitedCredits(t *testing.T) {
+	testcase := &acctest.PluginTestCase{
+		Name:     "amazon-ebs_unlimited_credits_test",
+		Template: testBurstableInstanceTypes,
+		Check: func(buildCommand *exec.Cmd, logfile string) error {
+			if buildCommand.ProcessState.ExitCode() != 0 {
+				return fmt.Errorf("Bad exit code. Logfile: %s", logfile)
+			}
+			return nil
+		},
+	}
+	acctest.TestPlugin(t, testcase)
+}
+
+//go:embed test-fixtures/unlimited-credits/burstable_spot_instances.pkr.hcl
```

Comment:
Good idea to leverage embed rather than trying to read it at runtime, we should probably migrate the existing tests that rely on that behaviour to use this instead. But that's for another PR